pages/docs/overview/start.md

```diff
@@ -9,19 +9,42 @@ toc: false
 ## Installation Quick Start
 
 ```bash
-git clone {{ site.repo }}.git
-cd singularity
-./autogen.sh
-./configure --prefix=/usr/local
-make
-sudo make install
+$ git clone {{ site.repo }}.git
+$ cd singularity
+$ ./autogen.sh
+$ ./configure --prefix=/usr/local
+$ make
+$ sudo make install
 ```
 
 ## Command Quick Start
 
+### Create a Centos7 image from a CentOS host
+
+It's easiest to build an image on a "compatible" host.  Here's a quick
+recipe to build a CentOS 7 image on a CentOS host.  See
+the [bootstrapping section][readme-bootstrapping] of the README for
+the full story and the [tutorials] for deeper advice.
+
+1. Create a file named `centos.def` that contains the following lines.
+
+   ```
+   BootStrap: yum
+   OSVersion: 7
+   MirrorURL: http://mirror.centos.org/centos-%{OSVERSION}/%{OSVERSION}/os/$basearch/
+   Include: yum
+   ```
+
```

Comment:
I really like this change to the Quickstart, but instead of doing Centos 7, could we do a technology that more users are familiar with, Docker? So provide an example of `Bootstrap: docker` and then maybe point the user to the folder of example specs for others.